libs/shared/common/src/lib/utilities/string.utils.ts

```diff
@@ -0,0 +1,7 @@
+const camelCaseToSentenceCase = (input: string) =>

```

Comment:
Add documentation

Reply:
Resolved in https://github.com/usamazansari/expenses-tracker/pull/62/commits/23dcbcd6064df9bcf23c155f20386b21e0cb95f8